### PR TITLE
add times related

### PR DESCRIPTION
**Pull request by fanweixiao**

### CODE REVIEW

File: time.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Time per item
```
Comment (everdale-wiki) [inline time.md]: The heading should be in title case.
```suggestion
# Time Per Item
```

File: time.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Villager's hunger time: 2m30s
```
Comment (everdale-wiki) [inline time.md]: This should be title case as well, along with an emphasis.
```suggestion
**Villager's Hunger Time:** 2m30s
```

**Review by everdale-wiki — CHANGES_REQUESTED** [review]

**Comment by everdale-wiki** [comment]
Thanks a lot for this pull request!

In addition to the above changes, could you please also reference this page in the [`README.md`](https://github.com/everdale-wiki/everdale-wiki.github.io/blob/main/README.md)? The `README.md` is the index page of the website and is the only way to navigate to other pages at the moment.

**Comment by fanweixiao** [comment]
Yes, I've added this page to the index in `README.md`.

**Pull request merged**